### PR TITLE
updates for rancher v2.6.11-rc1

### PR DESCRIPTION
**Pull request by geethub97**

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -161,9 +161,9 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     chmod +x /usr/bin/tini /usr/bin/telemetry && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
-ENV CATTLE_UI_VERSION 2.6.9
-ENV CATTLE_DASHBOARD_UI_VERSION v2.6.9
-ENV CATTLE_CLI_VERSION v2.6.9
+ENV CATTLE_UI_VERSION 2.6.11-rc1
+ENV CATTLE_DASHBOARD_UI_VERSION 2.6.11-rc1
```
Comment (prachidamle) [inline package/Dockerfile:165]: this should be: CATTLE_DASHBOARD_UI_VERSION v2.6.11-rc1 
The UI tag does not have the "v", but the Dashboard tag does

File: package/Dockerfile [hunk 2]
```diff
@@ -15,12 +15,12 @@ WORKDIR /var/lib/rancher
...
 ARG IMAGE_REPO=rancher
-ARG SYSTEM_CHART_DEFAULT_BRANCH=release-v2.6
-ARG CHART_DEFAULT_BRANCH=release-v2.6
+ARG SYSTEM_CHART_DEFAULT_BRANCH=dev-v2.6
+ARG CHART_DEFAULT_BRANCH=dev-v2.6
 ARG PARTNER_CHART_DEFAULT_BRANCH=main
 ARG RKE2_CHART_DEFAULT_BRANCH=main
 # kontainer-driver-metadata branch to be set for specific branch other than dev/master, logic at rancher/rancher/pkg/settings/setting.go
-ARG CATTLE_KDM_BRANCH=release-v2.6
+ARG CATTLE_KDM_BRANCH=dev-v2.6
```
Comment (prachidamle) [inline package/Dockerfile:23]: There is one more reference here we should update https://github.com/rancher/rancher/blob/release/v2.6/pkg/settings/setting.go#L59

Comment (prachidamle) [reply package/Dockerfile:23]: Also here https://github.com/rancher/rancher/blob/release/v2.6/Dockerfile.dapper#L15

**Review by prachidamle — APPROVED** [review]
LGTM

File: package/Dockerfile (hunk 2 above)
Comment (aiyengar2) [reply package/Dockerfile:23]: seems to be addressed

**Review by aiyengar2 — APPROVED** [review]
Seems like the inverse of https://github.com/rancher/rancher/pull/39581, so LGTM

**Pull request merged**